South_Africa/ZAF_transform_load_dlt.py

```diff
@@ -80,28 +73,17 @@ def boost_silver():
         .withColumn(
             "admin0",
             when(
-                (
-                    (col("admin0_temp") == "Municipalities")
-                    | (col("admin0_temp") == "Provinces")
-                ),
+                col("admin0_temp").isin("Provinces", "Municipalities"),
                 "Regional",
-            ).when((col("admin0_temp") == "Central"), "Central"),
+            ).otherwise("Central"),
         )
         .withColumn(
             "admin1",
-            when(col("admin0") == "Central", "Central Scope").when(
-                col("admin0") == "Regional", col("admin1")
-            ),
-        )
-        .withColumn(
-            "geo1",
-            when(col("admin0").isNull(), "Central Scope")
-            .when(
-                col("admin0") == "Central",
+            when(
+                col("admin0_temp").isin("Provinces", "Municipalities"),
```

Comment:
Consider `when(col('admin0') == 'Regional', col('Department'))` to avoid repeating the logic.